corgi/tasks/prod_defs.py

```diff
@@ -160,12 +179,10 @@ def parse_variants_from_brew_tags(brew_tags, name, product, product_stream, prod
     # I haven't filed a product bug to get them fixed since 6.7 - 6.9 are no longer
     # active. See CORGI-546 for more details.
     if (
-            len(brew_tags) > 0
-            and product_version.name != "quay-3"
-            and name
-            not in ["rhn_satellite_6.7", "rhn_satellite_6.8", "rhn_satellite_6.9"]
+        len(brew_tags) > 0
+        and product_version.name != "quay-3"
+        and name not in ["rhn_satellite_6.7", "rhn_satellite_6.8", "rhn_satellite_6.9"]
```

Comment:
Minor nit: use tuples for constant iterables:
```suggestion
        and name not in ("rhn_satellite_6.7", "rhn_satellite_6.8", "rhn_satellite_6.9")
```

Reply:
ok